graylog2-server/src/test/java/org/graylog/testing/containermatrix/ContainerMatrixTestEngine.java

```diff
@@ -98,12 +99,57 @@ private Set<Class<? extends PluginJarsProvider>> getPluginJarsProvider(Set<Class
         return get(annotatedClasses, (ContainerMatrixTestsConfiguration annotation) -> Stream.of(annotation.pluginJarsProvider()));
     }
 
-    private Set<SearchServer> getSearchServerVersions(Set<Class<?>> annotatedClasses) {
+    /**
+     * Property has to be in the form of Elasticsearch|OpenSearch-x.x.x
+     *
+     * @return
+     */
+    private static String getVersionOverride() {
+        return System.getProperty("graylog.matrix.tests.version.override");
```

Comment:
How about making the property name "search version" specific? If we want to do the same for MongoDB at some point, we would have a naming problem. :smile: 

```suggestion
        return System.getProperty("graylog.matrix.tests.search.version.override");
```